src/server/routes/nyplApiClient/index.js

```diff
@@ -1,27 +1,35 @@
 import NyplApiClient from '@nypl/nypl-data-api-client';
-import config from '../../../../appConfig.js';
 import aws from 'aws-sdk';
 
+import config from '../../../../appConfig.js';
+import logger from '../../../../logger.js';
+
 const appEnvironment = process.env.APP_ENV || 'production';
+const kmsEnvironment = process.env.KMS_ENV || 'encrypted';
 const apiBase = config.api[appEnvironment];
-const kms = new aws.KMS({
-  region: 'us-east-1',
-});
+let decryptKMS;
+let kms;
 
-function decryptKMS(key) {
-  const params = {
-    CiphertextBlob: new Buffer(key, 'base64'),
-  };
+if (kmsEnvironment === 'encrypted') {
```

Comment:
Yes, that's exactly what I did it the other way... It's fine, it'll only go into those blocks if the KMS env var is `encrypted` so we shouldn't run into issues.